src/v/config/configuration.cc

```diff
@@ -1545,7 +1546,46 @@ configuration::configuration()
       "Maximum capacity of rate limit accumulation"
       "in controller configuration operations limit",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
-      std::nullopt) {}
+      std::nullopt)
+  , kafka_throughput_limit_cluster_bps(
```

Comment:
> kafka_throughput_limit_cluster_bps(

i think we should drop this in this commit since it doesn't appear to be used anywhere.

---

separately let's have a discussion on slack around our plan for these configuration options. ultimately we want to have cluster-level throughput control, so this cluster-level configuration option makes sense.

but consider the case where we initially deliver a solution that has static quotas per node (but dynamic across shards). in this case, does it make more sense to have a per-node config option, or keep the cluster-level config option but dynamically assign a static value (e.g. quota / num_nodes)? the downside of the former is that it might mean unnecessary complexity dealing with configs when we later upgrade to having cluster-level quota smarts.

all indications i see suggest we don't need to let this decision block work on this PR nor the follow-up PR for cross-shard sharing since the "input" to the system is a single node quota value. we can continue to have this discussion async and adapt when its resolved.

Reply:
+1 to Noah's point.